mkdocs.yml

```
@@ -0,0 +1,59 @@
+site_name: Agent Factory
+site_url: https://mozilla-ai.github.io/agent-factory/
+repo_name: mozilla-ai/agent-factory
+repo_url: https://github.com/mozilla-ai/agent-factory
+docs_dir: docs
+
+extra_css: []
+
+theme:
+  name: material
+  palette:
+    - media: "(prefers-color-scheme: dark)"
+      scheme: slate
+      primary: black
+      accent: grey
+      toggle:
+        icon: material/weather-sunny
+        name: Switch to light mode
+    - media: "(prefers-color-scheme: light)"
+      scheme: default
+      primary: black
+      accent: grey
+      toggle:
+        icon: material/weather-night
+        name: Switch to dark mode
+  font:
+    text: Roboto
+    code: JetBrains Mono
+  features:
+    - navigation.instant
+    - navigation.sections
+    - toc.integrate
+
+markdown_extensions:
+  - admonition
+  - pymdownx.details
+  - pymdownx.superfences
+  - attr_list
+  - md_in_html
+  - pymdownx.blocks.caption
+  - toc:
+      permalink: true
+
+plugins:
+  - mkdocstrings:
+      default_handler: python
+      handlers:
+        python:
+          paths: [src]
+          options:
+            show_source: true
+            show_root_heading: true
+
+nav:
+  - Home: index.md
+  - Getting Started:
+    - Installation: getting-started/installation.md
+  - User Guide:
+    - MCP Servers: user-guide/mcp-servers.md
```

Just a suggestion re: the message we get when we run `make docs-build`:

```
INFO    -  The following pages exist in the docs directory, but are not included in the "nav" configuration:
             - user-guide/mcp-servers-template.md
```

As we do not want to include the template in the docs, we can explicitly exclude it to remove this message. Something like the following should work:

```suggestion
    - MCP Servers: user-guide/mcp-servers.md
    
exclude_docs: |
  user-guide/mcp-servers-template.md

```